PopupTabLayout/PopupTabLayout.module.less

```diff
@@ -80,6 +80,9 @@
 		.content .panels::before {
 			border-radius: @sand-popuptablayout-tabs-content-border-radius;
 			box-shadow: @sand-popuptablayout-shadow;
+			outline-color: @sand-overlay-outline-color;
+			outline-width: @sand-overlay-outline-width;
+			outline-style: @sand-overlay-outline-style;
```

Comment:
I think we missed outline-offset: -@sand-overlay-outline-width; here.

Reply:
Yep, I missed this. Thank you!